test/python/quantum_info/test_synthesis.py

```diff
@@ -206,6 +206,15 @@ def test_one_qubit_random_all_basis(self, basis, seed):
         unitary = random_unitary(2, seed=seed)
         self.check_one_qubit_euler_angles(unitary, basis)
 
+    @combine(basis=['U', 'PSX', 'ZSX', 'U3', 'U1X', 'ZYZ', 'ZXZ', 'XYX', 'RR'], seed=range(50),
+             name='test_one_qubit_global_phase_{basis}_basis_{seed}')
+    def test_one_qubit_global_phase_all_basis(self, basis, seed):
+        """Verify for {basis} basis and random_unitary (seed={seed})."""
+        unitary = random_unitary(2, seed=seed)
+        decomposer = OneQubitEulerDecomposer(basis)
+        decomp_unitary = Operator(decomposer(unitary))
+        self.assertTrue(np.allclose(unitary.data, decomp_unitary.data))
```

Comment:
Can you use `CheckDecompositions.check_one_qubit_euler_angles` here? It has a `phase_equal` flag that will include global phase. (Actually, it would be good to change the default there to `True`, since all of the synthesis methods tested here should be phase-correct.)

Reply:
Also, can you update `expand basis test.randomized.test_synthesis.TestSynthesis.test_1q_random` to include all of the bases tested here?

Reply:
I removed the test and set `phase_equal=True`. Randomized tests updated.